imports/api/pages/methods.js

```diff
@@ -68,3 +80,11 @@ function savePage (page, project) {
   Projects.update(project._id, { $set: githubProjectUpdate })
   return Pages.update(page._id, { $set: githubPageUpdate })
 }
+
+function uploadcareUuidFromUrl (uploadcareUrl) {
+  const invalidError = new Error(`Invalid Uploadcare URL: ${uploadcareUrl}`)
+  if (!uploadcareUrl) throw invalidError
+  const thisUrl = url.parse(uploadcareUrl)
+  if (thisUrl.hostname !== 'ucarecdn.com' || !thisUrl.pathname) throw invalidError
```

Comment:
Noted.  I think we probably need to update Social Probe and SohiWorld as well then...